include/zephyr/net/net_context.h

```diff
@@ -435,7 +435,7 @@ static inline void net_context_set_accepting(struct net_context *context,
 	if (accepting) {
 		context->flags |= NET_CONTEXT_ACCEPTING_SOCK;
 	} else {
-		context->flags &= ~NET_CONTEXT_ACCEPTING_SOCK;
+	  context->flags &= (uint16_t)~NET_CONTEXT_ACCEPTING_SOCK;
```

Comment:
should use tab for these

Reply:
Fixed

Reply:
[warnings.txt](https://github.com/zephyrproject-rtos/zephyr/files/14423686/warnings.txt)
Add the warnings generated from my build. If you want to reproduce the warning, try
git clone git@github.com:beechwoods-software/microROS-installer.git -b dave_patch_fix_with_pp_with_warnings ros_int; cd ros_int; ./docker_setup.sh
docker_setup.sh will set up a docker image and show you the build commands.